ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayDeviceLocationWithNmeaDataSources/README.md

```diff
@@ -12,25 +12,28 @@ The NMEA location data source allows for detailed interrogation of the informati
 
 ## How to use the sample
 
-Tap "Start" to parse the NMEA sentences into a simulated location data source, and initiate the location display. Tap "Reset" to reset the location display.
+Tap "Start" to parse the NMEA sentences into a simulated location data source, and initiate the location display. Tap "Recenter" to recenter the location display. Tap "Reset" to reset the location display.
 
 ## How it works
 
-1. Load NMEA sentences from a local file.
-2. Parse the NMEA sentence strings, and push data into `NmeaLocationDataSource`.
-3. Set the `NmeaLocationDataSource` to the location display's data source.
+1. Load an NMEA string and parse the sentences into a series of locations. ‼️ **(Desktop platform need more details about using GPS dongle)**
+2. Create an `AGSSimulatedLocationDataSource` and initialize it with the mock locations. ‼️ **(We can also directly parse the raw NMEA strings, and use `AGSNMEALocationDataSource` instead. Don't forget ti adjust the wordings)**
+3. Set it to the location display's data source.
 4. Start the location display to begin receiving location and satellite updates.
 
 ## Relevant API
 
-* LocationDisplay
-* NMEALocationDataSource
+* AGSLocation
+* AGSLocationDisplay
+* AGSNMEALocationDataSource
+* AGSNMEASatelliteInfo
+* AGSSimulatedLocationDataSource ‼️ remove this if you don't use a simulated data source
```

Comment:
```suggestion
* Location
* LocationDisplay
* NmeaLocationDataSource
```
We don't have an equivalent `AGSNMEASatelliteInfo` class yet, and this sample doesn't use a simulated location data source.